pkgs/development/python-modules/linkheader/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "LinkHeader";
+  version = "0.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hsh3gr8d5mc91c396xj33bv6zbfi0f7xnvi0m9vryx31dfc7fvz";
+  };
+
+  # Project has no tests
+  doCheck = false;
+  pythonImportsCheck = [ "link_header" ];
+
+  meta = with lib; {
+    description = "Python module to handle link headers according to RFC 5988";
+    homepage = "https://pypi.org/project/LinkHeader";
```

Comment:
This hasn't had an update for 5 year... and the upstream source no longer exists... I'm not sure it's a great candidate for nixpkgs inclusion.

Reply:
`LinkHeader` is an `extras_require` of `aiocoap`. In https://github.com/NixOS/nixpkgs/issues/62833 it was suggested to leave `extras_require`s out. I guess then it's fine to not include it.

Yes, it seems that upstream is dead.